client/src/components/_common/SystemsPushKeysModal/SystemsPushKeysModal.js

```diff
@@ -145,6 +139,11 @@ const SystemsPushKeysModal = () => {
                 />
               </ModalBody>
               <ModalFooter>
+                {error && !submitting && (
+                  <InlineMessage type="error" className="data-files-footer-msg">
```

Comment:
There's a new `SectionMessage` component that I think would be good to use here.

Reply:
Thank you. But, this one should be `<InlineMessage>`. There is recent propagation of new design pattern that _Errors in modal footers that occur from modal footer button action **both** use `<InlineMessage>` **and** be right-aligned._

- [FP-740](https://jira.tacc.utexas.edu/browse/FP-740) (via PR #265)
    - [Late Design - Action Success](https://jira.tacc.utexas.edu/secure/attachment/14723/Late%20Design%20-%20Action%20Success.png)
    - [Late Design - Confirm Action](https://jira.tacc.utexas.edu/secure/attachment/14722/Late%20Design%20-%20Confirm%20Action.png)
- https://github.com/TACC/Frontera-Portal/pull/264 (FP-369)

> Ideally, this design pattern is built into the `_common/Form` component, but that component is not full-featured, yet. The propagation of `<…Message>` is happening faster than I have bandwidth to document its usage.